plugins/analysis-icu/src/test/java/org/elasticsearch/index/analysis/SimpleIcuCollationTokenFilterTests.java

```diff
@@ -218,6 +218,21 @@ public void testCustomRules() throws Exception {
         TokenFilterFactory filterFactory = analysis.tokenFilter.get("myCollator");
         assertCollatesToSame(filterFactory, "Töne", "Toene");
     }
+    
+    /*
+     * Test a basic custom rules (should not interfere with reading rules list in IcuCollationTokenFilterFactory and throw InvalidPathException).
```

Comment:
nit: we have a line length limit to 140 characters so this needs to be broken up in order to pass the checkstyle test